components/configurator/view/SaveConfiguratorView.js

```diff
@@ -116,43 +120,49 @@ const SaveConfiguratorView = () => {
     default:
   }
 
+  let prompt
+  switch (step) {
+    case STEPS.READY:
+      prompt = (
+        <ConfiguratorPrompt message="ready?" actionA="sign" actionB="go back" />
+      )
+      break
+    case STEPS.SAVE:
+      prompt = <ConfiguratorPrompt message="saving..." />
+    case STEPS.PREMINT:
+      prompt = (
+        <ConfiguratorPrompt message="Minting coming soon..." actionA="mint" />
+      )
+      break
+    case STEPS.MINT:
+      prompt = <ConfiguratorPrompt message="minting..." />
+      break
+    case STEPS.CONFIRMATION:
+      prompt = (
+        <ConfiguratorPrompt
+          message="well done, your journey will begin soon..."
+          actionA="bring a friend"
+        />
+      )
+      break
+    case STEPS.ERROR:
+      prompt = (
+        <ConfiguratorPrompt
+          message={errorMessage}
+          actionA="try again?"
+          actionB="go back"
+        />
+      )
+      break
+  }
+
   return (
     <ConfiguratorContainer>
-      <ConfiguratorScreen>
-        {step === STEPS.READY ? (
-          <ConfiguratorPrompt
-            message="ready?"
-            actionA="sign"
-            actionB="go back"
-          />
-        ) : undefined}
-
-        {step === STEPS.SAVE ? (
-          <ConfiguratorPrompt message="saving..." />
-        ) : undefined}
-
-        {step === STEPS.PREMINT ? (
-          <ConfiguratorPrompt message="Minting coming soon..." actionA="mint" />
-        ) : undefined}
-
-        {step === STEPS.MINT ? (
-          <ConfiguratorPrompt message="minting..." />
-        ) : undefined}
-
-        {step === STEPS.CONFIRMATION ? (
-          <ConfiguratorPrompt
-            message="well done, your journey will begin soon..."
-            actionA="bring a friend"
-          />
-        ) : undefined}
-
-        {step === STEPS.ERROR ? (
-          <ConfiguratorPrompt
-            message={errorMessage}
-            actionA="try again?"
-            actionB="go back"
-          />
-        ) : undefined}
+      <ConfiguratorScreen withNav>
```

Comment:
We could adjust this area if we want, e.g. remove nav since you can't interact with it here but the nav is in the Figma designs so I made sure it was here too.

Reply:
👍